src/EditorManager.ts

```diff
@@ -65,7 +65,7 @@ export class CosmosEditorManager {
         }
 
         const localFilename = fileName.replace(/[<>:"\/\\|?*]/g, "-");
-        const localDocPath = path.join(os.tmpdir(), 'vscode-cosmosdb-editor', localFilename);
+        const localDocPath = path.join(os.tmpdir(), 'vscode-azuredatabases-editor', localFilename);
```

Comment:
> Should appended labels like this and this be changed

That feature is still specific to Cosmos DB, so I don't see a reason to change it.